builtin/gc.c

```diff
@@ -1345,6 +1345,154 @@ static int maintenance_task_incremental_repack(struct maintenance_run_opts *opts
 	return 0;
 }
 
+
+static void copy_file(const char *srcdir, const char *dstdir, const char *name)
```

Comment:
There's already `copy_file()` in `copy.h`, can we use that instead?

Reply:
Done!